packages/error-reporting/src/interfaces/message-builder.js

```diff
@@ -44,9 +45,7 @@ function handlerSetup(config) {
     //       error is used instead of the stack trace where the error is
     //       reported to be consistent with the behavior of reporting a
     //       an error when reporting an actual Node.js Error object.
-    var fauxError = new Error('');
-    var fullStack = fauxError.stack.split('\n');
-    var cleanedStack = fullStack.slice(2).join('\n');
+    var cleanedStack = buildStackTrace('', 1);
```

Comment:
I've updated `buildStackTrace` to use `Error.captureStackTrace`.  To remove all error-reporting frames would require restructuring the `src/interfaces/manual.js` file to expose the `reportManualError` function.  This would take more refactoring than I think is needed at this point.